android/app/src/main/java/org/uwaterloo/subletr/pages/home/list/HomeListChildView.kt

```diff
@@ -118,86 +130,203 @@ fun HomeListChildView(
+					) { closeBottomSheet(coroutineScope, scaffoldState) }
+
+					FilterType.ROOMS -> TODO()
+					FilterType.PROPERTY_TYPE -> TODO()
+					FilterType.ROOMMATE -> TODO()
+					FilterType.ALL -> TODO()
+				}
+
+
+			},
+			sheetSwipeEnabled = false,
+			sheetContainerColor = Color.White,
+			sheetPeekHeight = dimensionResource(id = R.dimen.zero),
+
+			) { padding ->
+			Scaffold(
+				modifier = Modifier
+					.padding(padding)
+					.fillMaxSize(1.0f),
+				floatingActionButtonPosition = FabPosition.End,
+				floatingActionButton = {
+					FloatingActionButton(
+						modifier = Modifier.padding(
+							all = dimensionResource(id = R.dimen.zero),
+						),
+						onClick = {
+							viewModel.navHostController.navigate(NavigationDestination.CREATE_LISTING.fullNavPath)
 						},
-						selectedValue = uiState.locationRange.stringId,
-						width = dimensionResource(id = R.dimen.xxxxl),
-					)
-					FilterDropDown(
-						filterName = stringResource(id = R.string.price),
-						dropDownItems = PriceRange.values().map { it.stringId }
-							.toTypedArray(),
-						updateState = { newVal ->
-							viewModel.priceRangeFilterStream.onNext(
-								PriceRange.fromInt(newVal)
+						shape = CircleShape,
+						containerColor = subletrPink,
+						contentColor = Color.White,
+					) {
+						Text(
+							stringResource(id = R.string.plus_sign), style = TextStyle(
+								fontSize = 24.sp
 							)
-						},
-						selectedValue = uiState.priceRange.stringId,
-						width = dimensionResource(id = R.dimen.xxxl),
-
 						)
-					FilterDropDown(
-						filterName = stringResource(id = R.string.rooms),
-						dropDownItems = RoomRange.values().map { it.stringId }
-							.toTypedArray(),
-						updateState = { newVal ->
-							viewModel.roomRangeFilterStream.onNext(
-								RoomRange.fromInt(newVal)
-							)
-						},
-						selectedValue = uiState.roomRange.stringId,
-						width = dimensionResource(id = R.dimen.xxxxl)
-					)
-				}
+					}
+				},
+				topBar = {
+					Box(modifier = Modifier)
+				},
+				bottomBar = {
+					Box(modifier = Modifier)
+				},
+			) { scaffoldPadding ->
+				LazyColumn(
+					modifier = modifier
+						.fillMaxSize(1.0f)
+						.padding(scaffoldPadding)
+						.padding(
+							dimensionResource(id = R.dimen.s),
+							dimensionResource(id = R.dimen.zero)
+						),
+					state = listState,
+					verticalArrangement = Arrangement.spacedBy(dimensionResource(id = R.dimen.s)),
+					horizontalAlignment = Alignment.CenterHorizontally,
+				) {
+					item {
+						LazyRow(
+							modifier = Modifier
+								.fillMaxWidth(1.0f),
+							verticalAlignment = Alignment.CenterVertically,
+							horizontalArrangement = Arrangement.spacedBy(dimensionResource(id = R.dimen.xs)),
+						) {
+							item {
+								ButtonWithIcon(
+									modifier = Modifier
+										.width(dimensionResource(id = R.dimen.xl))
+										.height(dimensionResource(id = R.dimen.l)),
+									iconId = R.drawable.tune_round_black_24,
```

Comment:
I know you didn't really change this in this PR but can you add a TODO to make this dark mode conscious?